plugins/modules/vault_pki_generate_certificate.py

```diff
@@ -232,7 +232,6 @@ def run_module():
 
     if not HAS_HVAC:
         module.fail_json(msg=missing_required_lib('hvac'), exception=HVAC_IMPORT_ERROR)
-        return
```

Comment:
Why remove this return ? To me module should be skip as quick as possible when import error is detected, that's the purpose of this return.